src/lean_spec/subspecs/networking/reqresp/message.py

```diff
@@ -27,6 +27,32 @@ class Status(StrictBaseModel):
     This is the first message sent upon a new connection and is essential for
     the peer-to-peer handshake. It allows nodes to verify compatibility and
     determine if they are on the same chain.
+
+    ***Enabled in Devnet 2***
+
```

Comment:
Right now we're not marking the fork diffs in the specs

```suggestion
```

Reply:
ohh right I mixed up devnet1 and devnet2, I thought @g11tech also gave an okay for this for devnet1. But I'll mark it for discussion tonight along with #137

Reply:
So in pqinterop call on Nov 12, it's clear I misunderstood and that this should go in devnet2 indeed.

I'm not sure how best to mark something that's slated for future devnets though. @tcoratger @fselmo?

Reply:
<img width="686" height="339" alt="image" src="https://github.com/user-attachments/assets/17fc5d37-835f-4b04-9f90-652da65c9f1f" />

@unnawut in this other part of code, they just say "For Devnet X" can I do that for this PR then merge this?